misk-hibernate/src/main/kotlin/misk/hibernate/Query.kt

```diff
@@ -79,3 +79,12 @@ annotation class Order(
   val path: String,
   val asc: Boolean = true
 )
+
+/**
+ * Annotates a function on an interface to execute a `SELECT` query. Functions with
+ * this annotation must return a `List` to fetch multiple rows results, or a regular type
```

Comment:
i'm confused by this javadoc. your example function returns a `QueryPlan`, not a result set